Celeste.Mod.mm/Mod/Core/CoreModuleSettings.cs

```diff
@@ -308,6 +308,56 @@ public bool ColorizedLogging {
         [SettingIgnore]
         public int DebugRCPort { get; set; } = 32270;
 
+        public bool PhotosensitiveMode {
+            get => Settings.Instance.DisableFlashes;
+            set => Settings.Instance.DisableFlashes = value;
+        }
+        
+        [SettingIgnore]
+        public bool PhotosensitivityDistortOverride { get; set; } = false;
```

Comment:
I still think having two public properties for each setting is very confusing. Ideally we should figure out a way to implement this while only exposing the Allow... properties (I'm convinced this is possible, YamlDotNet is just poorly documented unfortunately). Barring that, we should at least indicate that these shouldn't be used with something like the Obsolete attribute (though that doesn't quite fit).

Reply:
Addressed in commit [60e0aed](https://github.com/EverestAPI/Everest/pull/836/commits/60e0aed887428e74f89386f01090e3982a857437) using implementation proposed in [#modding-dev](https://discord.com/channels/403698615446536203/429775439423209472/1300537022900867094)